src/tests/unit/common/visualization-scan-result-store-data-builder.ts

```diff
@@ -53,6 +55,7 @@ export class VisualizationScanResultStoreDataBuilder extends BaseDataBuilder<IVi
     }
 
     public withSelectedIdToRuleResultMap(type: VisualizationType, selectedIdToRuleResultMap: any): VisualizationScanResultStoreDataBuilder {
+        // tslint:disable-next-line: switch-default
```

Comment:
are we okay with adding `tslint:disable` ?
I was also looking at fixing the `switch-default` issue but could not find a default value for these switch cases, so didn't try this! 😛

Reply:
The switch-default conversation is dancing around the fact that we really should be using a different design pattern here. This, every-method-has-an-enum-that-we-switch-on anti-pattern really should be addressed soon.

Reply:
This as now been removed